source/installation-guide/securing_api.rst

```diff
@@ -5,28 +5,97 @@
 Securing the Wazuh API
 ======================
 
-By default, the communications between the Wazuh Kibana App and the Wazuh API are not encrypted. It is highly recommended that you secure the Wazuh API by following the steps below:
+By default, the communications between the Wazuh Kibana plugin and the Wazuh API are not encrypted. In addition to this, the default user is "foo" and the password is  "bar". For this reason, it is highly recommended that you secure the Wazuh API and change the default user and password. The following steps will help you to achieve it:
```

Comment:
By default, the communication between the Wazuh Kibana plugin and the Wazuh API is not encrypted.

Reply:
Done: https://github.com/wazuh/wazuh-documentation/commit/bfc3e879e5b15d947be0460f676da6995bdc6e6e